<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextParametersSelectorClass</class>
 <widget class="QDialog" name="TextParametersSelectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextParametersSelector</string>
  </property>
  <widget class="QLabel" name="TextLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>251</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="FontLabel">
      <property name="text">
       <string>Font</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="SizeInput">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="SizeLabel">
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="FontInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="TextInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TextParametersSelector.qrc"/>
 </resources>
 <connections/>
</ui>
